declaration/templates/declaration/_declaration_form.html

```diff
@@ -0,0 +1,10 @@
+<div class="declaration-form">
+    <h1>Are you an HCP?</h1>
+    <div class="bag-image"></div>
+    <p><strong>I am</strong> a Healthcare professional</p>
+    <form action="{% url 'declaration:form' %}" method="post">
+        {% csrf_token %}
+        {{ form }}
+        <button type="submit">Yes</button>
```

Comment:
Looking at the rest of the wording `Yes` seems wrong and for completeness it should probably have a `name` and `value`:

```
<button type="submit" name="declaration" value="yes">I confirm I am a HCP</button>
```